webpack.config.js

```diff
@@ -46,7 +46,9 @@ module.exports = (env, argv) => {
         // application to productions standards
         nodeEnv = "production";
     }
-    const devMode = nodeEnv !== 'production';
+    // Properly set NODE_ENV for CI
+    process.env.NODE_ENV = process.env.NODE_ENV ?? nodeEnv;
```

Comment:
This can probably go, it's automatically set by the `mode` entry

```suggestion
```

Reply:
we are not using `mode` in the webpack config, maybe that's why this entire problem occured.

Reply:
`mode` doesn't set process.env.NODE_ENV.